taxii2client/v21/__init__.py

```diff
@@ -99,9 +99,9 @@ def _populate_fields(self, id=None, status=None, total_count=None,
         self.success_count = success_count  # required
         self.failure_count = failure_count  # required
         self.pending_count = pending_count  # required
-        self.successes = successes or []  # optional
-        self.failures = failures or []  # optional
-        self.pendings = pendings or []  # optional
+        self.successes = successes or None  # optional
+        self.failures = failures or None    # optional
+        self.pendings = pendings or None    # optional
```

Comment:
Technically these don't need the `or None` at all any more, but handling it like this also deals with situations when the element might be otherwise falsey and result in confusion. I've not seen that occur though, so happy to simplify this if something things it's pointless.